README.md

```diff
@@ -25,3 +23,37 @@ ds = open_dataset(
     branch="my_new_testdata_branch"
 )
 ```
+
+> **Note**
+> The following options only work for branches based on `Ouranosinc/xclim-testdata`, not forks.
+
+If you wish to run the entire `xclim` testing suite locally against your branch, this can be set via an environment variable:
+```shell
+$ export XCLIM_TESTDATA_BRANCH="my_new_testdata_branch"
+
+$ pytest xclim
+# or, alternatively:
+$ tox
+```
+
+If you wish to run the entire `xclim` testing suite on the `Ouranosinc/xclim` GitHub Workflows (CI) against your branch,
+this can be set via an environment variable default in the `.github/workflows/main.yml` workflow configuration:
+```yaml
+env:
+  XCLIM_TESTDATA_BRANCH: my_new_testdata_branch
+```
+
+> **Note**
+> Be aware that modifying this variable to a value other than the latest tagged version of `xclim-testdata`
+> will trigger a GitHub Workflow that will block merging of your Pull Request until changes are effected.
+
+## Versioning
+When updating a dataset in `xclim-testdata` using a development branch and Pull Request, once changes have been merged
+to the `main` branch, you should tag a new version of `xclim-testdata`. The version tag of `xclim-testdata` is loosely tied
+to that of `xclim`, and should represent the `xclim` release (**stable**, i.e. version string not ending in `-dev`)
+that is compatible with the latest changes in `xclim-testdata`.
+
+> **Warning** 
+> In the event that multiple Pull Requests occur for `xclim-testdata` between two stable versions of `xclim`
+> (e.g. one for `v1.2.3-dev` and another for `v1.2.5-dev`), the `xclim-tesdata` tag (e.g. `v1.3.0`)
```

Comment:
This entire block of comment unclear to me.

Are we trying to say "all PR to `xclim-testdata` repo should be cumulative"?  The newer PR should include changes from all previous PR even if those previous PR not yet merged?

Reply:
This is probably the weakest part of the schema, since it relies on us deleting a tag if two PRs are merged during a stable version.

I'm trying to say that in the case of a PR being merged, a new tagged commit being created, then another PR being merged soon after (between stable `xclim` releases), the previous tag needs to be replaced with the newest changes.

Reply:
Here's a case example:

1. Changes are made in a PR in `xclim` (Occurring for `xclim @ v1.2.3-dev`) that require changes to `xclim-tesdata`
2. The developer creates a new branch of `xclim-testdata` for their PR at `xclim` (e.g. `my_new_testdata`) and tells `xclim` CI (`main.yml`) to use this branch for testing.
3.  (A GitHub Actions workflow is triggered to prevent merging this development branch to `xclim`'s `master` branch, but otherwise lets tests continue.
4. Changes are approved in their development branch at `xclim`.
5. The `xclim-testdata` branch is merged to `main`, and a new tag is made to reflect the**minimum stable version** of `xclim` required (in this case, `xclim @ v1.3.0`).
6. (The developer updates the branch of `xclim-testadata` to test against in their `xclim` dev branch. GitHub Actions checks to see if the tag is the newest; All good = branch no longer blocked).
7. Developer merges their PR to `xclim` main branch. Everyone rejoices.

Now the issue is that if this occurs again for another PR in `xclim` before `v1.3.0` is released, we **need** to replace the `xclim-testdata` tag. This is doable, but not ideal. I'm not sure how else to make things work.

Reply:
> Now the issue is that if this occurs again for another PR in `xclim` before `v1.3.0` is released, we **need** to replace the `xclim-testdata` tag. This is doable, but not ideal. I'm not sure how else to make things work.

How about detach the matching tags between `xclim` and `xclim-testdata`?

Might even just use date as tags on `xclim-testdata` so make it clear there is no relation between the tags.

Then there can be more than 1 new tags on `xclim-testdata` between the stable releases of `xclim`.

Reply:
That's a much better idea. Honestly, I don't think this case will ever occur, but having a date-based versioning system is much nicer. I can implement that on the `xclim` side next week, after the release. Thanks for the idea!